playground/react/src/routes.tsx

```diff
@@ -7,17 +7,21 @@ import LoginPage from './pages/LoginPage';
 const RouteProvider = () => {
   const routes = [
     {
-      path: '/auth',
+      path: '/auth/:projectId',
```

Comment:
Why didn't you follow the implementation idea here? I think its more logical to have a structure where the projectId comes first.

I think that it's easier to structure the routes like this (then we still have our normal route structure that we had before, we just inserted the projectId):
/pro-XXX/auth
/pro-XXX/login
/pro-XXX/register
/pro-XXX/

instead of having this:
/auth/pro-XXX
/register/pro-XXX
/login/pro-XXX
/pro-XXX